oca_dependencies.txt

```diff
@@ -2,3 +2,5 @@
 # Add a repository url and branch if you need a forked version
 account-analytic
 web
+account_analytic_parent https://github.com/minorisa/account-analytic.git 11.0-mig-account_analytic_parent
```

Comment:
Is it https://github.com/OCA/account-analytic/tree/11.0/account_analytic_parent?